assets/base.css

```diff
@@ -447,7 +456,7 @@ blockquote {
 .caption {
   font-size: 1rem;
   letter-spacing: 0.07rem;
-  line-height: 1.7;
+  line-height: calc(1 + 0.7 / var(--font-body-scale));
```

Comment:
**Side note, not related to this PR**: I find this element's font size literally impossible to read on mobile. And my phone has a huuuge almost table-sized screen, so I can't even imagine how hard it is on smaller devices (`1rem` = `10px`)

Is this intentional?